components/WithdrawLiquidity/WithdrawLiquidityView.tsx

```diff
@@ -158,29 +157,13 @@ const WithdrawLiquidity = ({
             </div>
           </div>
 
-          <div className={style.totalPriceDiv}>
-            <div className={style.priceTag}>
-              <p>Price</p>
-            </div>
-            <div className={style.totalPrice}>
-              <p>
-                1 {token0.symbol || '-'} = 4295.45 {token1.symbol || '-'}
-              </p>
-              <p>
-                1 {token0.symbol || '-'} = 0.000232804 {token1.symbol || '-'}
-              </p>
-            </div>
-          </div>
-
-          <div className={style.totalPriceDiv}>
-            <div className={style.priceTag}>
-              <p>Balance in Liquidity</p>
-              <p>Balance in {token1 ? token1.symbol : ''}</p>
-            </div>
-            <div className={style.totalPrice}>
-              <p>{liquidityAmt}</p>
-              <p>{liqInTokB}</p>
-            </div>
+          <div className={style.totalPriceContainer}>
+              <div className={style.unitTokenConversion}>
+                4295.45 {token1.symbol || '-'} = 1 {token0.symbol || '-'}
```

Comment:
Why is this 4295.45 value hard coded?

Reply:
This value is hardcoded becaues currently we do not know what 1 TokenA is equal to TokenB

Reply:
hmm.. this value should come from the uniswap pair. Put // TODO: once factory contract is available it should be the price that comes from the uniswap pair.

Reply:
Ok